org.gnome.Boxes.Extension.OsinfoDb.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id" : "org.gnome.Boxes.Extension.OsinfoDb",
+    "branch" : "master",
```

Comment:
```suggestion
    "branch" : "master",
```

Reply:
sorry, should this be "stable" too? It is unclear to me whether this refer to the flatpak branching or the osinfo-db git branch name.

Reply:
This is flatpak branch, Flathub automation sets it automatically based on git branch. Usually there's no need to specify it.